appveyor.yml

```diff
@@ -1,8 +1,8 @@
 environment:
   ELM_VERSION: 0.18
   matrix:
-  - nodejs_version: "5.0"
-  - nodejs_version: "4.2"
+    - nodejs_version: "5.0"
+    - nodejs_version: "4.2"
```

Comment:
I might be wrong, but those node versions are no longer supported by this package. 🤔

Reply:
this even looks like an old config, presumably not in use. references elm 18. perhaps remove it?

Reply:
I'll revert the formatting changes, just to keep this MR about getting the dependencies upgraded